mlflow/models/evaluation/validation.py

```diff
@@ -68,16 +69,18 @@ def __init__(
                 raise MetricThresholdClassException(
                     "`min_relative_change` parameter must be between 0 and 1."
                 )
-        if higher_is_better is None:
-            raise MetricThresholdClassException("`higher_is_better` parameter must be defined.")
-        if not isinstance(higher_is_better, bool):
-            raise MetricThresholdClassException("`higher_is_better` parameter must be a boolean.")
+        if higher_is_better is None and greater_is_better is None:
+            raise MetricThresholdClassException("`greater_is_better` parameter must be defined.")
+        if greater_is_better is None:
+            greater_is_better = higher_is_better
+        if not isinstance(greater_is_better, bool):
+            raise MetricThresholdClassException("`greater_is_better` parameter must be a boolean.")
```

Comment:
Can we raise an exception if both `higher_is_better` and `greater_is_better` are specified?